content/cucumber/api.md

```diff
@@ -763,13 +747,88 @@ Cucumber does not work when installed globally because cucumber needs to be requ
 
 You can also run features using a [build tool](/tools/general#build-tools) or an [IDE](/tools/general#ides).
 
-# Configuration
 
-Cucumber provides several configuration options.
+## JUnit
```

Comment:
JUnit section probably should be moved to Java tools page. We could link to it here. (This might be outside the scope of the PR; if you want to do it, please do. Otherwise, create an issue and we'll do it later)

Reply:
Np. Will do